clients/imodels-client-authoring/src/operations/FileDownload.ts

```diff
@@ -41,7 +42,10 @@ export async function downloadFile(params: DownloadFileParams): Promise<void> {
     });
     downloadStream.pipe(targetFileStream);
 
-    if (params.totalDownloadCallback || params.latestDownloadedChunkSizeCallback){
+    if (
+      params.totalDownloadCallback ||
+      params.latestDownloadedChunkSizeCallback
+    ) {
```

Comment:
is this manual formatting? I know clients repo should use prettier to manage line length  but we currently do not and we do not have steps in pipelines to ensure formatting. My point is that we should follow this formatting but this is another PR and it is hard to review what has actually changed. Please revert the formatting

Reply:
Updated